### ISSUE TITLE
Totals do not add up

### ISSUE THREAD

**Issue by mgdatas** [CITF-Malaysia/citf-public#164]
Hello and thank you to all those working on providing this valuable data to the public.

I notice that the 'cumul' figures are not equal to ('cumulative_partial' + cumulative_full') nor ('cumulative_partial' + cumulative_full' + 'cumulative_booster'). May I know how I can obtain the relationships between the different data?

Thank you.

**Comment by KennethThePro** [comment]
`daily_partial` is consist of the 1st doses (for double-dose vaccines) only
`cumul_partial` is the sum of both `daily_partial` and `cansino`

`daily_full` consist of 2nd doses (for single-dose vaccines) and 1-dose vaccines (e.g. Cansino)
`cumul_full` is the sum of `daily_full`

`daily_booster` consist of all booster shots
`cumul_booster` is the sum of `daily_booster`

To obtain the total cumulative doses administered, it may see like a straight forward answer which is
 `cumul_partial` + `cumul_full` + `cumul_booster`

However, single-dose vaccine like Cansino is defined in both `cumul_partial` and `cumul_full` which means that there is a duplicate count. Therefore the final formula to obtain the `cumul` is as follow
`cumul` =  `cumul_partial` + `cumul_full` + `cumul_booster` - cumulative of cansino

**Comment by mgdatas** [comment]
Thank you for the clarification, KennethThePro. Appreciate it. I will check
and get back to you if necessary.

May I communicate with you directly via this email address?

Just one more clarification pertaining to your response above. Should this
statement: daily_full consist of 2nd doses (for *single-dose* vaccines) and
1-dose vaccines (e.g. Cansino) actually read as:
 daily_full consist of 2nd doses (for *double-dose* vaccines) and 1-dose
vaccines (e.g. Cansino)?

Thank you again and regards.

Mohan G

**Comment by KennethThePro** [comment]
Yes, my mistake should had been double dose vaccines, will edit the comment
